changelog.md

```diff
@@ -1,3 +1,121 @@
+
+## CGA
+* Changes to existing features:
+  * comp operation: New component selector fv for face vertices.
+  * inline(unify), union, subtract, intersect keywords: Boolean operations keep general vertex order.
+  * union, subtract, intersect keywords: Boolean operations set a new auto-tag "bool.cut" at intersection edges.
+* Bugfixes:
+  * inline, union, subtract, intersect keywords: Fixed the material of the resulting shapes for some cases where non-default materials were involved.
+  * convexify operation: Fixed a crash that could happen when small edges are present.
+  * For some operations the first edge of the resulting geometry was made more consistent and deterministic. Since the first edge often determines the scope setup this can lead to different results of existing rules.
+    * innerRectangle operation: The first edge of remainder faces is set at the outline of the initial face. The order of remainders is set wrt. the vertex order of the initial face. Before, the behavior was arbitrary.
+    * setback, setbackToArea, setbackPerEdge, shapeLUO, splitAndSetbackPerimeter operations: In rare cases the first edge of setback faces was not set at those edges selected by the setback selector. Further, the first edge of setback faces is now set only at those edges with a positive setback distance (>0). Finally, we fixed an issue where first edges and order of remainder faces were arbitrary in the presence of holes.
+    * envelope operation: Fixed an issue where first edges of slanted faces were not aligned to the outline of the initial shape when holes are present.
+    * i, trim operations: Trimming at concave boundary keeps the first edge, if not trimmed away.
+  * comp operation, comp function: Fixed numerical unstable orientation of horizontal edges when using comp(e) with logical selector expressions or static selectors eave, hip, valley, and ridge.
+
+## Built-In Codecs
+* Added Decoder for the TGA image format (com.esri.prt.codecs.TGADecoder).
+* Added basic support for reading and writing CityGML files (com.esri.prt.codecs.CityGMLDecoder, com.esri.prt.codecs.CityGMLEncoder)
+* Collada (DAE) Decoder:
+  * Support for files with animations. Animations are ignored. Before such files could not be read at all. 
+  * Fixed a bug where the transparency was set to a wrong value.
+* Collada, GLTF, FBX, IFC, DWG, USD Decoders:
+  * Added support for "/scaling/appliedUnitToMeter" metadata
+* IFC decoder:
+  * Added support for "/georef/" nmetdata (IFC 4).
```

Comment:
```suggestion
  * Added support for "/georef/" metdata (IFC 4).
```

Reply:
haha, there is still a typo. I'll fix it.